apps/experimental/e2b-test/src/full-benchmark.ts

```
@@ -0,0 +1,298 @@
+// Claude reads the result to verify
+await step("Read: verify login.tsx exists", async () => {
+  await sandbox.files.read("/home/user/project/src/login.tsx")
+})
+
+// Grep for the import
+await step("Grep: verify import in index.ts", async () => {
+  await sandbox.commands.run("grep -n 'loginRouter' /home/user/project/src/index.ts")
+})
+
+const totalSession = steps.reduce((sum, s) => sum + s.ms, 0)
+console.log(`\n  ─────────────────────────────────────────────────────`)
```

Biome (2.4.4)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)